packages/scxa-cell-type-wheel-experiment-heatmap/cypress/component/CellTypeWheelExperimentHeatmap.cy.js

```diff
@@ -43,10 +43,8 @@ describe(`CellTypeWheelExperimentHeatmap`, () => {
     />)
     const ringElement = cy.contains(`regulatory T cell`)
     ringElement.click({ force: true })
-    cy.contains(`H2-Q7`)
-    cy.contains(`Cd3g`)
-    cy.contains(`Ltb`)
-    cy.contains(`Trac`)
-    cy.contains(`Shisa5`)
+    heatMapTCellResponse.map((cell) => cell.geneName).forEach(
+      geneName => cy.contains(geneName)
+    )
```

Comment:
I think this one is a bit easier to read:
```js
heatMapTCellResponse.forEach(cell => cy.contains(cell.geneName))
```

If you go with your version I think you can do:
```js
    heatMapTCellResponse.map((cell) => cell.geneName).forEach(cy.contains)
```

Reply:
We can use the 1st if the `geneName` is always there.
Changed.